### DISCUSSION TITLE
winget doesn't work for me when installing from appinstaller

### DISCUSSION THREAD

**Discussion by aayushkdev** [microsoft/winget-cli#1041] [Q&A] [answered]
hey so winget doesn't work for me when I try installing it from appinstaller 
I have signed up from http://aka.ms/winget-InsiderProgram and got the link ms-windows-store://pdp/?productid=9nblggh4nns1 but the link doesn't update app installer 
I have also tried solutions from https://github.com/microsoft/winget-cli/issues/210 but had no success

so does anyone have any solutions for this

**Comment by Josh65-2201** [comment, marked as answer]
Note that this will only work on windows 10 1809 or later. If you meet that try this link `ms-appinstaller:?source=https://aka.ms/getwinget` just put the whole line into your browser URL box. it worked for me.

**Reply by aayushkdev** [reply]
thanks it worked for me
